rst_files/linear_algebra.rst

```diff
@@ -278,35 +264,29 @@ The expression :math:`\| x - y\|` is thought of as the distance between :math:`x
 Continuing on from the previous example, the inner product and norm can be computed as
 follows
 
+.. code-block:: julia
 
-
+    using LinearAlgebra
 
 
 .. code-block:: julia
 
     dot(x, y)               # Inner product of x and y
 
 
-
 .. code-block:: julia
 
-    sum(x .* y)             # Gives the same result
+    sum(i * j for (i, j) ∈ zip(x, y))             # Gives the same result
```

Comment:
This is way too verbose.  I would rather have an unnecessary allocation then something so complicated.

Reply:
```
sum(*, zip(x, y)) # errors out
```
but this works
```
sum(prod, zip(x, y))
```